reactions/src/HALO.cpp

```diff
@@ -355,8 +349,9 @@ return result;
 // real
 double HALO::rvirial(double lgmass, double vars[]) const {
       //virial overdensity
+      double omegacb = vars[1]-vars[6];
       double Deltavir = (1.+ delta_avir(lgmass))*pow3(vars[0]/a_vir(lgmass));
-      return 0.1*pow((pow(10.,lgmass)*Gnewton)/(5.*vars[1]*Deltavir), 1./3.);
+      return 0.1*pow((pow(10.,lgmass)*Gnewton)/(5.*omegacb*Deltavir), 1./3.);
```

Comment:
can you reference an equation here (what is 0.1?)

Reply:
Have added a comment. The 0.1 pops out upon converting rho_m to the correct units.